client/commonFramework/show-completion.js

```diff
@@ -68,6 +68,70 @@ window.common = (function(global) {
       });
   }
 
+  function submitGithubHandler(e) {
+    e.preventDefault();
+
+    var solution = common.editor.getValue();
+
+    $('#save-challenge')
+      .attr('disabled', 'true')
+      .removeClass('btn-primary')
+      .addClass('btn-warning disabled');
+
+    var $checkmarkContainer = $('#checkmark-container');
+    $checkmarkContainer.css({ height: $checkmarkContainer.innerHeight() });
+
+    $('#challenge-checkmark')
+      .addClass('zoomOutUp')
+      // .removeClass('zoomInDown')
+      .delay(1000)
+      .queue(function(next) {
+        $(this).replaceWith(
+          '<div id="challenge-spinner" ' +
+          'class="animated zoomInUp inner-circles-loader">' +
+          'submitting...</div>'
+        );
+        next();
+      });
+
+    let timezone = 'UTC';
+    try {
+      timezone = moment.tz.guess();
+    } catch (err) {
+      err.message = `
+          known bug, see: https://github.com/moment/moment-timezone/issues/294:
```

Comment:
Starting from this point you break the indentation style of the repo. You've used 4 spaces instead of 2. Needs to be fixed in all your files.